src/browser/findDOMNode.js

```diff
@@ -36,8 +34,8 @@ function findDOMNode(componentOrElement) {
     return ReactMount.getNodeFromInstance(componentOrElement);
   }
   invariant(
-    !(componentOrElement.render != null && typeof(componentOrElement.render) === 'function'),
-    'Component contains `render` method but is not mounted in the DOM',
+    componentOrElement.render == null || typeof(componentOrElement.render) !== 'function',
```

Comment:
I obviously like the demorganized. Also, this is > 80 chars, can you split at the `||` to quiet one more lint complaint.